bindings/cpu_profiler.cc

```diff
@@ -112,13 +112,10 @@ v8::Local<v8::Object> CreateSample(const uint32_t stack_id, const int64_t sample
   return js_node;
 };
 
-std::string hashCpuProfilerNodeByPath(const v8::CpuProfileNode* node) {
-  std::string path = std::string();
-  std::string delimiter = std::string(";");
-
+std::string delimiter = std::string(";");
```

Comment:
nit: `const auto delimeter = ...` 

maybe even give it a prefix to indicate it's a constant (e.g. `kDelimeter`)